package.json

```diff
@@ -8,7 +8,7 @@
     "start": "bin/node-raylib examples/core/core_basic_window.js",
     "benchmark": "bin/node-raylib examples/textures/textures_bunnymark.js",
     "test": "jest",
-    "posttest": "standard tools test examples index.js",
+    "posttest-ignore": "standard tools test examples index.js",
```

Comment:
Need to restore the `standard` test once the tests pass.

Reply:
`standard --fix` can be helpful, but I also use it in my editor for automatic fix-on-0save